src/routes/routes.js

```diff
@@ -0,0 +1,23 @@
+import App from '../App';
+import { Home, Categories } from '../pages';
+
+const { createBrowserRouter } = require('react-router-dom');
+
+const router = createBrowserRouter([
+  {
+    path: '/',
+    element: <App />,
+    children: [
+      {
+        index: true,
+        element: <Home />,
+      },
+      {
+        path: 'categories',
+        element: <Categories />,
+      },
+    ],
+  },
+]);
+
+export default router;
```

Comment:
- [ ] @Stephen-Adom great work on your however the project requirements indicate that routes should be done in the App.js component as shown below
![image](https://user-images.githubusercontent.com/61408860/228890900-ef9b6369-a13f-4a70-b0d9-741f190b1e3f.png)